config.properties

```diff
@@ -0,0 +1,4 @@
+browser = chrome
+env = http://qa2.trycloud.net/
+username = User9
```

Comment:
I think in properties file is conflict in username and password because you have scenario outline